vuepress/docs/next/tutorials/getting-started/kubernetes-install.md

```diff
@@ -2,45 +2,47 @@
 
 ## Infrastructure Requirements
 
-These are the minimum infrastructure requirements to install Entando directly on a Kubernetes (K8s) cluster:
+Below are the minimum infrastructure requirements to install Entando directly on a Kubernetes (K8s) cluster:
 
-- A working DNS server configured to resolve internal and external names. If you choose to configure K8s with a global wildcard domain name for the entire cluster, be sure it points to the public IP address of the external Load Balancer of the worker nodes.
-- A default [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is needed for the Entando Operator to create the correct Persistent Volume Claims (PVC) for each pod.
-- Nginx [Ingress-Controller](https://kubernetes.github.io/ingress-nginx/deploy/) is needed to expose the Entando applications.
+- A working DNS server configured to resolve internal and external names. To configure K8s with a global wildcard domain name for the entire cluster, be sure it points to the public IP address of the external load balancer for the worker nodes.
+- A default [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is needed for the Entando Operator to create the correct persistent volume claims (PVC) for each pod.
+- NGINX [ingress controller](https://kubernetes.github.io/ingress-nginx/deploy/) is needed to expose the Entando Application.
+- To publish bundles that do not contain Docker images to Git, the cluster containing the Entando Application must have access to the bundle repository, which can be public or private.
```

Comment:
Let's call this out as 2 items:
* To install bundles, the cluster containing the Entando Application must have access to the bundle repository (either public or private, often on GitHub) as well as the Docker image repository (e.g. DockerHub) for any microservices.

FYI, for 7.1, the new bundle style will be to draw only from Docker, but I believe the current style will still be supported so this requirement won't immediately change